.flake8

```diff
@@ -1,21 +1,18 @@
 [flake8]
 min_python_version = 3.6.0
-ignore =
-    # E501: Line too long
+max-line-length = 88
+inline-quotes = double
+extend-ignore =
+    # E501: Line too long (FIXME: long string constants)
     E501,
-    # W503: Line break occurred before a binary operator
-    W503
+    # E203: Whitespace before ':' (pycqa/pycodestyle#373)
+    E203,
 per-file-ignores =
-    # F401: Module imported by unused
+    # F401: Module imported by unused (non-implicit modules)
     __init__.py:F401
-exclude =
-    */fixtures/**/setup.py
-    .git,
-    .github,
-    .mypy_cache,
-    .pytest_cache,
-    .tox,
-    .venv,
-    .vscode,
-    __pycache__,
-    dist,
+    # INP001: File is part of an implicit namespace package.
+    get-poetry.py:INP001,
+    install-poetry.py:INP001,
```

Comment:
I would exclude these files altogether as these should be really frozen now.

Reply:
Sounds good.